kolibri/core/assets/src/vue/nav-bar/index.vue

```diff
@@ -91,7 +91,6 @@
 
   .nav-wrapper
     display: table
-    table-layout: fixed
```

Comment:
you wanna keep this to sure in portrait mode, each nav item takes the same width.